src/utils/build_importer.py

```diff
@@ -0,0 +1,116 @@
+from logger import Logger
+from pathlib import Path
+import os
+import re
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from yaml import dump
+
+
+def import_build():
+    Logger.info(f"Paste maxroll.gg build guide or planner build here ie https://maxroll.gg/d4/build-guides/minion-necromancer-guide")
+    url = input()
+    if "maxroll.gg/d4/planner" not in url and "maxroll.gg/d4/build-guides/" not in url:
+        Logger.info(f"Invalid url, please use a d4 planner build on MaxRoll.gg")
+        return
+    options = webdriver.ChromeOptions()
+    options.add_argument("--headless=new")
+    driver = webdriver.Chrome(options=options)
+    driver.get(url)
+    wait = WebDriverWait(driver, 10)
+    wait.until(EC.element_to_be_clickable((By.CLASS_NAME, "ncmp__btn")))
```

Comment:
Starting from here, lots of magic strings. For maintainability I would suggest moving them to module level as constants.

I'm referring to class names, xpaths, etc. Basically everything you use to locate the elements